dashboards-observability/public/components/trace_analytics/components/common/filters/filter_helpers.tsx

```diff
@@ -58,7 +59,7 @@ const getType = (field: string): string => {
     startTime: 'date_nanos',
   };
   const type = _.get(typeMapping, field, 'keyword');
-  return typeof type === 'string' ? type : null;
+  return type;
```

Comment:
why this change?

Reply:
I thought line 61 wasn't necessary because the typeof type is always a string by definition

Reply:
not exactly, if `field` is `attributes`, `type` is an object. But I forgot what was the use case here, you can keep this change and revert if anything breaks

Reply:
Ohh that makes sense. No I'll put it back.